ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -353,30 +327,13 @@ instance SupportsTwoPhaseValidation (BabbageEra c) where
 instance SupportsTwoPhaseValidation (ConwayEra c) where
   isIncorrectClaimedFlag _ = \case
     SL.ConwayUtxowFailure
-      ( Babbage.AlonzoInBabbageUtxowPredFailure
-          ( Alonzo.ShelleyInAlonzoUtxowPredFailure
-              ( SL.UtxoFailure
-                  ( Babbage.AlonzoInBabbageUtxoPredFailure
-                      ( Alonzo.UtxosFailure
-                          ( Conway.ValidationTagMismatch
-                              (Alonzo.IsValid _claimedFlag)
-                              _validationErrs
-                            )
-                        )
-                    )
-                )
-            )
-        ) -> True
-    SL.ConwayUtxowFailure
-      ( Babbage.UtxoFailure
-          ( Babbage.AlonzoInBabbageUtxoPredFailure
-              ( Alonzo.UtxosFailure
```

Comment:
Were these unified on `Conway.UtxoFailure`?

Reply:
I think this is answered by this ticket, right? https://github.com/IntersectMBO/cardano-ledger/issues/4170

Reply:
Yeah, there is also this in the commit message
```
  * Update `SupportsTwoPhaseValidation` instance for `Conway`:
    `ConwayUtxowFailure` was fixed to get rid of the possibility
    of injecting `UtxoFailure` in two separate ways
    (as seen in the code removed by this commit)
```